atest/acceptance/keywords/waiting.robot

```diff
@@ -52,6 +63,16 @@ Wait Until Page Does Not Contain Element
     ...    Custom Error ää ÖÖ
     ...    Wait Until Page Does Not Contain Element    content    0.1 seconds    Custom Error ää ÖÖ
 
+Wait Until Page Does Not Contain Element
```

Comment:
1) Using locator that matches more than one element could  be better test.
2) Testing with zero value would be interesting.

Reply:
Test cases added